packages/ERTP/test/unitTests/test-issuerObj.js

```diff
@@ -129,6 +129,22 @@ test('issuer.makeEmptyPurse', async t => {
     .then(checkWithdrawal);
 });
 
+test('purse.withdraw overdrawn', async t => {
+  t.plan(1);
+  const { issuer, mint, brand } = makeIssuerKit('fungible');
+  const purse = issuer.makeEmptyPurse();
+  const purseBalance = AmountMath.make(brand, 103980n);
+  const payment = mint.mintPayment(purseBalance);
+  purse.deposit(payment);
+
+  const tooMuch = AmountMath.make(brand, 103981n);
+
+  t.throws(() => purse.withdraw(tooMuch), {
+    message:
+      'Withdrawal of {"brand":"[Alleged: fungible brand]","value":"[103981n]"} failed because the purse only contained {"brand":"[Alleged: fungible brand]","value":"[103980n]"}',
```

Comment:
At this version, I see you’re using `assert`, you’re using `X`, and you’re not using `q`. I believe all of those are correct decisions. I am _confused_ that the test case passes, since it requires all the redacted details to be visible here.

I think I need to check whether there’s a ses-ava/SES bug now by reproducing this problem in isolation.

Reply:
This test passes in isolation:
```js
import 'ses';
import './lockdown.js';
import rawTest from 'ava';
import { wrapTest } from '@endo/ses-ava';

const test = wrapTest(rawTest);

test('redaction is redacted', t => {
  t.throws(() => {
    assert.fail(assert.details`This should be redacted: ${{secret: 42}}`);
  }, {
    message: 'This should be redacted: (an object)',
  });
});
```
So, I remain confused if this test is still passing.

Reply:
Perhaps amount has a toString behavior that’s getting called automatically?

@katelynsills I think it would be absolutely fair to change this test to a regexp so that it passes regardless of whether the amount gets redacted, since you’re not testing SES or ses-ava here.

Reply:
I had to phone a friend (@erights) but I’m now satisfied that the test works as expected because this test is run with `SwingSet/tools/install-ses-debug.js` which sets `errorTaming: 'unsafe'` so redactions are deliberately disabled. This PR is fine as stands!